lib/service_check.pm

```diff
@@ -231,6 +231,9 @@ sub install_services {
     my ($service) = @_;
     # turn off lmod shell debug information
     assert_script_run('echo export LMOD_SH_DBG_ON=1 >> /etc/bash.bashrc.local');
+    # On ppc64le, sometime the console font will be distorted into pseudo graphics characters.
+    # we need to reset the console font. As it impacted all the console services, added this command to bashrc file
+    assert_script_run('echo /usr/lib/systemd/systemd-vconsole-setup >> /etc/bash.bashrc.local') if check_var('ARCH', 'ppc64le');
```

Comment:
is this a bug or expected behaviour? In both cases it would be good to have an external reference, e.g. a bug report, that confirms this

Reply:
Reported one bug before, but developer didn't think this is a bug. This one just miss some configuration, Setup systemd-vconsole-setup can fix this problem. This also isn't expected behaviour.

Reply:
Refer bug: https://bugzilla.suse.com/show_bug.cgi?id=1180307

Reply:
@okurz Do you have any comment about this PR?

Reply:
sorry, I do not understand where it was mentioned that this is not a valid bug. https://bugzilla.suse.com/show_bug.cgi?id=1180307 however explains that others could not yet reproduce it manually. https://bugzilla.suse.com/show_bug.cgi?id=1180307 has been set as RESOLVED FIXED but I am not sure, was there an actual fix then?

Reply:
According comment #11: Did I get it right and it works correctly if you do a manual invocation to systemd-vconsole-setup like here?
https://openqa.nue.suse.com/tests/5206886#step/install_service/12
We just follow this suggestion to fix problem

Reply:
Yes, but "this prevents the problem" is not really an explanation if this is expected behaviour. Anyway, it's ok. Don't mind me :)

Reply:
@okurz， Can we add this workaround at here?

Reply:
Yes, as I stated: Don't mind me. Just keep it as you added it, it's fine